jujugui/static/gui/src/app/components/terminal/terminal.js

```diff
@@ -22,10 +22,15 @@ class Terminal extends React.Component {
   constructor(props) {
     super(props);
     this.state = {
-      size: 'min'
+      size: 'min',
+      terminalSize: null
```

Comment:
Difference between size and terminalSize? Maybe add a comment?

Reply:
done